flow-typed/bip32.js

```diff
@@ -0,0 +1,31 @@
+declare module 'bip32' {
+  declare class BIP32 {
+    depth: number;
+    parentFingerprint: number;
+    index: number;
+    publicKey: Buffer;
+    privateKey: Buffer;
+    chainCode: Buffer;
+    network: Network;
+    fingerprint: Buffer;
+    identifier: Buffer;
+    static fromBase58(
+      str: string,
+      networks: ?(Array<Network> | Network)
+    ): BIP32;
+    static fromSeed(seed: Buffer, network?: ?Network): BIP32;
+    getPublicKeyBuffer(): Buffer;
```

Comment:
As far as I can tell, this method doesn't actually exist.

Reply:
looks like what you want is `.publicKey` ?